src/views/GovernanceReviewTeam/AllRequests.tsx

```diff
@@ -77,10 +103,36 @@ const AllRequests = () => {
     headerGroups,
     rows,
     prepareRow
-  } = useTable({
-    columns,
-    data
-  });
+  } = useTable(
+    {
+      columns,
+      data,
+      initialState: {
+        sortBy: [{ id: 'submittedAt', desc: true }]
+      }
+    },
+    useSortBy
+  );
+
+  const getHeaderSortIcon = (isDesc: boolean | undefined) => {
+    return classnames('margin-left-1', {
+      'fa fa-caret-down': isDesc,
+      'fa fa-caret-up': !isDesc
+    });
+  };
+
+  const getColumnSortStatus = (
```

Comment:
This is a great purpose-built function! Would love to see some unit tests for the various permutations here at some point.